lib/src/local_working_copy.rs

```diff
@@ -1091,7 +1119,7 @@ impl TreeState {
         repo_path: &RepoPath,
         disk_path: &Path,
         current_tree_values: &MergedTreeValue,
-        executable: bool,
+        executable: FileExecutableFlag,
```

Comment:
nit: perhaps, this belongs to the first commit in the stack.

Reply:
Oops, good catch. Fixed.